packages/jest-coverage/src/index.ts

```diff
@@ -0,0 +1,77 @@
+/**
+ * Copyright (c) Facebook, Inc. and its affiliates. All Rights Reserved.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+import inspector from 'inspector';
+
+type ScriptCoverage = inspector.Profiler.ScriptCoverage;
+export type Coverage = ReadonlyArray<ScriptCoverage>;
+
+export default class CoverageInstrumenter {
+  private readonly session = new inspector.Session();
+
+  async startInstrumenting() {
+    this.session.connect();
+
+    await new Promise((resolve, reject) => {
```

Comment:
I lost type info if I used `util.promisify`, so I just wrapped everything in promises manually

Reply:
Thanks for looking into it! I don't think it's worth the complexity here, but maybe you could upstream those changes to DT? The definitions are here: https://github.com/DefinitelyTyped/DefinitelyTyped/blob/796b838a15fad73287bad7a88707a9ca04e60640/types/node/util.d.ts#L86-L105

Reply:
I completely agree, I think this actually belongs to DT's codebase since it's essentially covering what the type defs should be there.

I'll have a look at that, thanks @SimenB 😊